Strata/DDM/Util/Deser.lean

```diff
@@ -159,36 +161,36 @@ protected def peekM' (act : BufferM α) : AReader α := fun f => do
   return (← act, f.unchanged)
 
 instance : MonadReader ByteArray AReader where
-  read := .peekM' .contents
+  read := private .peekM' .contents
 
 protected def curOffset : AReader Nat :=
   .peekM fun f => return (← .contents).size - f
 
 def canRead (len : Nat) : AReader Bool :=
-  .peekM fun f => pure (f >= len)
+  .peekM fun f => .pure (f >= len)
 
 protected def skip! (len : Nat) : AReader Unit := fun f => do
-  pure ((), ⟨f - len, Fuel.sub_le f len⟩)
+  .pure ((), ⟨f - len, Fuel.sub_le f len⟩)
 
 protected def skip (off : Nat) (len : Nat) : AReader Unit := fun f => do
   if f ≥ len then
-    pure ((), ⟨f - len, Fuel.sub_le f len⟩)
+    .pure ((), ⟨f - len, Fuel.sub_le f len⟩)
```

Comment:
Could you use `return` and `throw` keyword here? If the AReader implements a monad, it would make maintenance less expensive.